dts/bindings/sram/sifive,dtim0.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2019 Antmicro <www.antmicro.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Data Tighly Integrated Memory
+version: 0.1
+
+description: >
+    This binding gives description of the SiFive Data Tighly Integrated Memory
```

Comment:
Minor change: "This binding describes the SiFive Data Tightly-Integrated Memory"

Reply:
Fixed.